pvlib/temperature.py

```diff
@@ -599,7 +599,8 @@ def fuentes(poa_global, temp_air, wind_speed, noct_installed, module_height=5,
     # n.b. the way Fuentes calculates the first timedelta makes it seem like
     # the value doesn't matter -- rather than recreate it here, just assume
     # it's the same as the second timedelta:
-    timedelta_hours = np.diff(poa_global.index).astype(float) / 1e9 / 60 / 60
+    timedeltas = (poa_global.index[1:] - poa_global.index[:-1])
```

Comment:
is this equivalent to `poa_global.index.to_series().diff().total_seconds() / 3600`?

Reply:
The type is different (series vs index), but that doesn't matter in this context.  With `timedelta_hours.iloc[0] = timedelta_hours.iloc[1]`, the two are equivalent, so I'll use yours :+1: